### PR TITLE
also print imported age keys

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: pkgs/sops-install-secrets/main.go [hunk 1]
```diff
@@ -534,15 +534,18 @@ func importAgeSSHKeys(keyPaths []string, ageFilePath string) error {
...
 			return fmt.Errorf("Cannot convert ssh key '%s': %w", p, err)
 		}
 		// Append it to the file
-		_, err = ageFile.WriteString(*bech32 + "\n")
+		_, err = ageFile.WriteString(*privKey + "\n")
 		if err != nil {
 			return fmt.Errorf("Cannot write key to age file: %w", err)
 		}
+
+		fmt.Printf("%s: Imported %s as Age key with fingerprint %s\n", path.Base(os.Args[0]), p, *pubKey)
```
Comment (Mic92) [inline pkgs/sops-install-secrets/main.go:548]: I also managed to encrypt secrets with shamir encryption by having the wrong .sops.yaml configuration.
So having this log message is actually quite helpful for debugging.

**Comment by dasJ** [comment]
Not a huge fan of adding more output since #113 adds up to 3 lines of output. What do you think about only doing that on boot (→ when /run/secrets does not exist yet)?

**Comment by Mic92** [comment]
> Not a huge fan of adding more output since #113 adds up to 3 lines of output. What do you think about only doing that on boot (→ when /run/secrets does not exist yet)?

This does not help really, users that migrate to sops, won't reboot their machines for testing.
If we don't have this than it  quite a pain to debug if users forgot to rotate keys after updating .sops.yaml
This is also relevant if users open up issues posting their yaml files and boot logs.

**Comment by dasJ** [comment]
What about an environment variable like `SOPS_SILENT_KEY_IMPORT`? I could add another one to #113 like `SOPS_SILENT_CHANGE_DETECTION`, then everybody can fit the behaviour to their liking

**Comment by dasJ** [comment]
I have implemented something similar but based on the manifest instead of the environment: https://github.com/Mic92/sops-nix/pull/113/commits/6473b947edad6d112ea7747f783e7f0219197d91

**Comment by Mic92** [comment]
I think I want to only print imported keys when an error happens. This should please all parties and requires no option. Than I can also make error messages better when gnupgHome or age.keyFile is used.

**Pull request closed without merging**